config/experiment.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+//Experiment defines if experimental features are available
+type Experiment struct {
+	AdCerts ExperimentAdsCert `mapstructure:"adscert"`
+}
+
+//ExperimentAdCerts configures and enables functionality to generate and send Ads Cert Auth header to bidders
+type ExperimentAdsCert struct {
+	Enabled   bool      `mapstructure:"enabled"`
+	InProcess InProcess `mapstructure:"in-process"`
+	Remote    Remote    `mapstructure:"remote"`
+}
+
+//InProcess configures data to sign requests using ads certs library in core PBS logic
+type InProcess struct {
```

Comment:
Nitpick: Please rename to `AdsCertInProcess` since this is in the shared config package.